src/components/Buildbaron/BBComponents.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { StyledLink } from "components/styles";
+import { getDateCopy } from "utils/string";
+
+interface titleProps {
+  margin?: boolean;
+}
+
+export const BBTitle = styled(Subtitle)<titleProps>`
+  margin-bottom: ${(props) => (props.margin ? "15px" : "5px")};
+  margin-top: ${(props) => (props.margin ? "25px" : "20px")};
+  // font-size: 16px;
+  line-height: 24px;
+  font-weight: bold;
+`;
+
+export const JiraTicketRow: React.FC<{
+  jiraKey: String;
+  fields;
```

Comment:
What is the type for fields?

Reply:
![image](https://user-images.githubusercontent.com/13104717/93898997-da534780-fcc1-11ea-8b17-f223500cb22c.png)